test/test_01_01.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from hm_1.tasks.task_01_01 import check_power_of_2
+
+
+@pytest.mark.parametrize("value", [0, 1, 4, 9, 16, 64, 144, 65536])
```

Comment:
`2 ** X == 0` please solve this for X

Reply:
As I understand, Such X doesn't exist. And it means that sqrt from 0 doesn't exist too and we need to consider this case as false?

Reply:
You just proved that 0 is not a power of two, why do you ask me what should be don't with it?